src/megameklab/com/util/UnitUtil.java

```diff
@@ -4438,4 +4437,31 @@ public static void removeEmptyBays(Entity entity) {
                 .collect(Collectors.toList());
         emptyBays.forEach(bay -> UnitUtil.removeMounted(entity, bay));
     }
+
+
+    public static void addProtomechAmmo(Protomech entity, EquipmentType ammo, int shots) throws LocationFullException {
```

Comment:
```suggestion
    public static void addProtoMechAmmo(Protomech entity, EquipmentType ammo, int shots) throws LocationFullException {
```
I know most of our internals have this issue, but can we please try to start fixing this typo?